examples/view_clearcontrol_dataset.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import tempfile
+import time
+
+from iohub.clearcontrol import (
+    ClearControlFOV,
+    create_mock_clear_control_dataset,
+)
+
+
+def main():
+    """Example of opening a Clear Control dataset and viewing with napari.
+
+    Usage
+    -----
+
+    ```
```

Comment:
Markdown syntax may not get rendered properly by sphinx (which uses rst).

Speaking of which, do you want to add this new API to the documentation page?

Reply:
Got it, fixed and added it to the API docs

Reply:
Sorry my previous comment was wrong/confusing. These should actually either be under `Examples` section and use the doctest syntax ([NumPy docs style guide](https://numpydoc.readthedocs.io/en/latest/format.html#examples)) or be the CLI help message. But not a big deal since we'll need to migrate these to the web page hopefully soon.